gm4_apple_trees/data/gm4_fruiting_trees/functions/load.mcfunction

```diff
@@ -0,0 +1,7 @@
+execute if score gm4 load matches 1 if score gm4_orbis load matches 1 run scoreboard players set gm4_fruiting_trees load 1
+execute unless score gm4 load matches 1 run data modify storage gm4:log queue append value {type:"missing",module:"Apple Trees",require:"Gamemode 4"}
+execute unless score gm4_orbis load matches 1 run data modify storage gm4:log queue append value {type:"missing",module:"Apple Trees",require:"Orbis"}
```

Comment:
These checks should either be removed or moved to gm4_apple_trees:load, to avoid duplicate errors.

Reply:
oh.. we forgot to change the names. Those are supposed to say "Fruiting Trees", not "Apple Trees"

Reply:
But fruiting trees and apple trees are supposedly always packaged together. Only the actual name of the module should announce errors.